### PR TITLE
Report messages

### PR DESCRIPTION
**Pull request by ChristophWurst**
Ref https://github.com/nextcloud/server/pull/15475

- [x] Requires Nextcloud 17 https://github.com/ChristophWurst/nextcloud_sentry/pull/117
- [x] Adjust for https://github.com/ChristophWurst/nextcloud_sentry/pull/112

### CODE REVIEW

File: lib/Command/Test.php [hunk 1]
```diff
@@ -48,7 +48,8 @@ protected function configure() {
 	public function execute(InputInterface $input, OutputInterface $output) {
 		$this->logger->info("staring sentry test command");
```
Comment (MorrisJobke) [inline lib/Command/Test.php]:
```suggestion
		$this->logger->info("starting sentry test command");
```

**Comment by MorrisJobke** [comment]
> [ ] Requires Nextcloud 17

Then also change the `info.xml` and keep this PR open

File: lib/Command/Test.php [hunk 2]
```diff
@@ -46,9 +46,10 @@ protected function configure() {
 	}
 
 	public function execute(InputInterface $input, OutputInterface $output) {
-		$this->logger->info("staring sentry test command");
+		$this->logger->info("starting sentry test command");
```
Comment (rullzer) [inline lib/Command/Test.php]: I prefer `staring`. Makes the app sound like a movie start :P

Comment (ChristophWurst) [reply lib/Command/Test.php]: well but then I would have to be "starring" right?

**starring Sentry the test command**

**Pull request merged**